README.md

```diff
@@ -111,6 +111,14 @@ Bots also will respond to a direct message, with or without the bot name in the
 
 ![](screenshots/dms.gif)
 
+### Logger Level Configuration
+By default Bots set a Logger to STDOUT and displays all Log levels. If you want to  change the logging level for your Production Bot the level should be changed as follow:
```

Comment:
Lots of nitpicks ;)

Add a space above.

Bots and Log should probably not be capitalized. I would say something more specific about which level the logger is on (DEBUG, INFO?) vs. "all log levels", that's not true. 

I wouldn't say "Production", that's not a characteristic of the bot, maybe just remove it and say something about changing bot logger level.

There's a double space between "to" and "change".